### PR TITLE
Add instructions for Linux.

### PR DESCRIPTION
**Pull request by jeremymeng**

### CODE REVIEW

File: docs/GettingStartedLinux.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    dnvm upgrade -u
+```
+
+The `-u` switch uses the unstable DNX feed, which is default to
+<https://www.myget.org/F/aspnetvnext/api/v2>.
+
+We can install a CoreCLR runtime by running the following command
+
+```
+    dnvm install 1.0.0-beta5-11649 -u -r coreclr -arch x64
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: Should this be beta5-*?

Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: There seems to be a bug in dnvm that doesn't allow this

File: docs/GettingStartedLinux.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+to get DNVM and run
+
+```
+    dnvm upgrade -u
+```
+
+The `-u` switch uses the unstable DNX feed, which is default to
+<https://www.myget.org/F/aspnetvnext/api/v2>.
+
+We can install a CoreCLR runtime by running the following command
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: I needed to run 'sudo apt-get install unzip' to get dnvm to work with coreclr

File: docs/GettingStartedLinux.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Use your favorite web browser to navigate to `http://localhost:5004`.
+
+If you want to use a runtime installed on your machine, just modify
+the path of `dnx` command in `~/site/Kestrel`.  For example, use the
+following command to remove the bundled CoreCLR runtime, add a shared DNX
+runtime installed under `~/.dnx` to `$PATH`.
+
+```
+    rm -rf ~/site/approot/packages/dnx-coreclr-linux-x64.1.0.0-beta5-11649/
+    dnvm use 1.0.0-beta5-11649r coreclr -arch x64
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: typo: `dnvm use 1.0.0-beta5-11649 -r coreclr -arch x64`

File: docs/GettingStartedLinux.md (hunk 2 above)
Comment (jeremymeng) [inline docs/GettingStartedLinux.md]: It's a bug in asp.net vnext docs that they didn't list `unzip` as a requirement.

File: docs/GettingStartedLinux.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+like
+
+```
+    Active Version              Runtime Arch Location             Alias
+    ------ -------              ------- ---- --------             -----
+      *    1.0.0-beta5-11649    coreclr x64  ~/.dnx/runtimes      default
+           1.0.0-beta5-11649    mono         ~/.dnx/runtimes
+```
+
+##Get the PartsUnlimited Code##
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: This is unnecessary

File: docs/GettingStartedLinux.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+Clone the PartsUnlimited repo.  The URL is
+<https://github.com/Microsoft/PartsUnlimited>.
+Checkout the `dev/beta5` branch
+
+
+##Develop##
+
+Switch to use a Mono-based DNX runtime. The version can be picked
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: You should call out that it is currently not supported to restore/publish with coreclr

File: docs/GettingStartedLinux.md (hunk 1 above)
Comment (jeremymeng) [inline docs/GettingStartedLinux.md]: We need to pass `<semver>|<alias>|<nupkg>|latest` to install command.

File: docs/GettingStartedLinux.md (hunk 5 above)
Comment (jeremymeng) [inline docs/GettingStartedLinux.md]: Added.  It's also mention at the top of the doc.

File: docs/GettingStartedLinux.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+Then you can use commands like `dnu build`, `dnu pack`, `dnu publish`,
+etc., on your ASP.Net vNext projects.
+
+For example, to publish the PartsUnlimited website app to `~/site`
+
+```
+    cd src/PartsUnlimitedWebsite
+    dnu restore
```
Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: Before you do this, you need to `sudo apt-get install nodejs` and install grunt

Comment (jeremymeng) [inline docs/GettingStartedLinux.md]: Yeah, I missed the following

```
sudo apt-get install npm nodejs
sudo npm install bower -g
sudo npm install grunt-cli -g
```

Comment (twsouthwick) [inline docs/GettingStartedLinux.md]: Can you add those line, then I"ll merge it in.

**Pull request merged**